templates/portal/base.html.twig

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```

Comment:
Is this the same file as base.html.twig?

Reply:
This file is not needed. I'll delete it.